config/routes.rb

```diff
@@ -124,12 +124,12 @@
         edit_step :ufn
         edit_step :case_contact
         edit_step :client_detail
+        upload_step :primary_quote
```

Comment:
yeah you're right - I was keeping it in there in case we needed the ability to delete the uploaded file but the file is compulsory so agreed with Mary it should just be an overwrite